### PR TITLE
humann v 3.6.1

### PR DESCRIPTION
**Pull request by tuncK**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

I revisited humann and it looks like it still works after the new DB release.

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

**Review by bernt-matthias — COMMENTED** [review]
There have been already some changes in https://github.com/galaxyproject/tools-iuc/pull/4403 .. maybe you can pick those up / continue from there?

File: tools/humann/humann_barplot.xml [hunk 1]
```diff
@@ -142,13 +142,13 @@ humann_barplot
         </section>
     </inputs>
     <outputs>
-        <data name="barplot_png" format="png" from_work_dir="output.png">
+        <data name="barplot_png" format="png" from_work_dir="output.png" label="png figure from ${tool.name} on ${on_string}">
```
[the remark this answers is not in the document]
Comment (tuncK) [reply tools/humann/humann_barplot.xml:145]: > There have been already some changes in #4403 .. maybe you can pick those up / continue from there?

@bernt-matthias We already have merged an alternative to it (and hence already have 3.6.0 in the repo), can you please close #4403?

[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/humann/humann_barplot.xml:145]: Sure. It's hard to keep an overview :)

[the remark this answers is not in the document]
Comment (tuncK) [reply tools/humann/humann_barplot.xml:145]: > Please stick to the naming scheme: `${tool.name} on ${on_string}: ...`

I actually do not have any reason to change it, but a linting error is forcing me to eliminate duplicate labels (and they are considered duplicate if they do not exist).

File: tools/humann/humann_barplot.xml [hunk 2]
```diff
@@ -142,13 +142,13 @@ humann_barplot
...
             <filter>out['format'] == 'png'</filter>
         </data>
-        <data name="barplot_pdf" format="pdf" from_work_dir="output.pdf">
+        <data name="barplot_pdf" format="pdf" from_work_dir="output.pdf" label="pdf figure from ${tool.name} on ${on_string}">
```
Comment (bernt-matthias) [inline tools/humann/humann_barplot.xml:148]:
```suggestion
        <data name="barplot_pdf" format="pdf" from_work_dir="output.pdf" label="${tool.name} on ${on_string}: pdf">
```

File: tools/humann/humann_barplot.xml [hunk 3]
```diff
@@ -142,13 +142,13 @@ humann_barplot
...
             <filter>out['format'] == 'pdf'</filter>
         </data>
-        <data name="barplot_svg" format="svg" from_work_dir="output.svg">
+        <data name="barplot_svg" format="svg" from_work_dir="output.svg" label="svg figure from ${tool.name} on ${on_string}">
```
Comment (bernt-matthias) [inline tools/humann/humann_barplot.xml:151]:
```suggestion
        <data name="barplot_svg" format="svg" from_work_dir="output.svg" label="${tool.name} on ${on_string}: svg">
```

**Pull request merged**